classes/local/skel/version_php_file.php

```diff
@@ -37,6 +37,20 @@
  */
 class version_php_file extends php_internal_file {
 
+    /** @var string[] Moodle versions. */
+    protected static $moodleversions = [
```

Comment:
Made the attribute static so I can send use it in the static function get_template_variables as a list of accepted values for the variable required.